lnprototest/errors.py

```diff
@@ -20,7 +20,27 @@ def path_to_str(self) -> str:
             result += f"{event},"
         return f"{result}]"
 
+    @staticmethod
+    def decode_hex_data(hex_data: str) -> str:
+        """Decode hex data into readable text if possible"""
+        try:
+            # Try to decode as ASCII/UTF-8
+            decoded = bytes.fromhex(hex_data).decode("utf-8", errors="replace")
+            # If the decoded text is mostly readable, return it
+            if all(ord(c) < 128 for c in decoded):
+                return f" (decoded: {decoded})"
+        except (ValueError, UnicodeDecodeError):
+            pass
```

Comment:
This is really dangerous, please do not try to shoot yourself in the foot when you do not need it, we are already using Python.